sdk/react/widgets/VideoViewPlayPause.js

```diff
@@ -112,15 +114,15 @@ class VideoViewPlayPause extends React.Component {
     );
   };
 
-  _renderSeekButton = (name, iconScale) => {
-    if (!this.props.showSeekButtons) {
+  _renderSeekButton = (name, iconScale, active) => {
+    if (!this.props.showSeekButtons || !this.props.seekEnabled) {
       return <View/>
     }
     const fontStyle = {fontSize: this.props.fontSize * iconScale, fontFamily: this.props.icons[name].fontFamily};
     const sizeStyle = {width: this.props.buttonWidth, height: this.props.buttonHeight};
     const opacity = {opacity: this.state.skipButtons.animationOpacity};
     const animate = {transform: [{scale: this.state.skipButtons.animationScale}]};
-    const buttonColor = {color: !!this.props.buttonColor ? this.props.buttonColor : "white"};
+    const buttonColor = {color: active ? !!this.props.buttonColor ? this.props.buttonColor : "white" : "gray"};
```

Comment:
Please don't nest turnary statements. They are hard to follow. Can you separate out this logic into if statements and then assign buttonColor the result.

Reply:
changed it